### ISSUE TITLE
Element.classes.add() not Unimplemented Exception

### ISSUE THREAD

**Issue by tom-clik** [dint-dev/universal_html#28]
The example in "#3 Use" won't compile and throws an UnimplementedError

dependencies:
  universal_html: ^1.2.1

Unhandled exception:
UnimplementedError

#0      _ElementCssClassSet.add (package:universal_html/src/html/dom/css_class_set.dart:435:5)
#1      main (file:///D:/codebase/dart/test1/cli/bin/cli.dart:27:21)
#2      _startIsolate.<anonymous closure> (dart:isolate-patch/isolate_patch.dart:299:32)
#3      _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.dart:168:12)

**Comment by 0xVesion** [comment]
I'm facing the same issue:

```
Dart SDK version: 2.10.4 (stable) (Wed Nov 11 13:35:58 2020 +0100) on "macos_x64"
```

```
dependencies:
  universal_html: ^1.2.4
```

```
Unhandled exception:
UnimplementedError
#0      _ElementCssClassSet.add (package:universal_html/src/html/dom/css_class_set.dart:435:5)
#1      main (---)
#2      _startIsolate.<anonymous closure> (dart:isolate-patch/isolate_patch.dart:301:19)
#3      _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.dart:168:12)
```

**Comment by terrier989** [comment]
Thanks for the issue report! This is fixed in v2.0.2.
